dashboard/app/models/peer_review.rb

```diff
@@ -30,14 +30,53 @@ class PeerReview < ActiveRecord::Base
   belongs_to :level
   belongs_to :level_source
 
+  after_save :mark_user_level
+
   REVIEWS_PER_SUBMISSION = 2
+  REVIEWS_FOR_CONSENSUS = 2
 
   enum status: {
```

Comment:
@philbogle recommended the Rails built-in enum type.  It seems like a good fit for this use case.